common/src/main/java/com/wynntils/models/abilities/ShamanTotemModel.java

```diff
@@ -47,6 +53,9 @@ public class ShamanTotemModel extends Model {
 
     private final ShamanTotem[] totems = new ShamanTotem[MAX_TOTEM_COUNT]; // 0-indexed list of totems 1-4
     private final Integer[] pendingTotemVisibleIds = new Integer[MAX_TOTEM_COUNT]; // ID of totems that need a timer
+    private final Map<Integer, Integer> orphanedTimers =
+            new ConcurrentHashMap<>(); // IDs of timers that can't really find a totem
```

Comment:
Why a concurrent map? From what I see, this field is only accessed from a single event/method (with it being `TextDisplayChangedEvent.Text`). Maybe this was left over from some alternative change?